api/cl_khr_command_buffer.asciidoc

```diff
@@ -4,15 +4,15 @@
 include::{generated}/meta/{refprefix}cl_khr_command_buffer.txt[]
 
 // *Revision*::
-//    0.9.6
+//    0.9.7
```

Comment:
This is revision 0.9.8.

Can we just remove this commented out text, though?  It's all auto-generated now.